src/components/teampage/team-members/team-member-styles.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components"
+import { size } from "../../../constants/devices"
+
+const CardPosition = styled.div`
+  min-width: calc(50% - 30px);
+  height: 300px;
+  display: inline-flex;
+  margin-bottom: ${props =>
+    props.isLast && props.index % 2 === 0 ? "0" : "8%"};
+  margin-left: ${props => (props.index % 2 === 0 ? "0" : "60px")};
+  position: relative;
+
+  @media only screen and (max-width: ${size.tablet}px) {
+    display: flex;
+    min-width: 100%;
+    margin: 0;
+    height: initial;
+  }
+`
+const ContactCardContainer = styled.div`
+  display: inline-flex;
+  min-width: 100%;
+  position: absolute;
+  top: ${props => (props.index % 2 === 0 ? "0" : "180px")};
+  left: ${props => (props.position === "Executive Director" ? "25%" : 0)};
+  @media only screen and (max-width: ${size.tablet}px) {
+    position: initial;
+    margin-bottom: 40px;
+    flex-direction: ${props => (props.index % 2 === 0 ? "row" : "row-reverse")};
+  }
+`
+
+const GraidentSquare = styled.div`
+  width: 50%;
+  padding-top: 50%;
+  position: absolute;
+  top: 40px;
+  background: linear-gradient(147.64deg, #c8dcee 19.39%, #546bdf 94.2%);
+  @media only screen and (max-width: ${size.tablet}px) {
+    display: none;
+  }
+`
+
+const InfoContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+
+  padding: 0 4%;
```

Comment:
Minor but the spacing doesn't completely align for mobile (ie: Megha's info is pushed farther left than the pics of Heenal and Peiying). You can probably make this be 2% for mobile or use the padding constant from `devices.js`
<img src="https://user-images.githubusercontent.com/60420625/116790480-5bf22b80-aa82-11eb-9fde-65346c464745.jpeg" alt="team page spacing" width="300" />